### ISSUE TITLE
Allow Jamstats to connect to a scoreboard via wsproxy

### ISSUE THREAD

**Issue by dhmay** [dhmay/jamstats#131]
It should be possible for Jamstats to connect to a scoreboard elsewhere on the internet whose traffic is being forwarded using [wsproxy](https://github.com/DerbyStats/wsproxy). That would allow remote announcers access to scoreboard data.

**Comment by dhmay** [comment]
wsproxy is written in Go, which I'm not familiar with. @DerbyStats, who wrote wsproxy, says it uses exactly the same protocol as the scoreboard.

I've tried to make this work locally. When I give wsproxy this config:

listen_address=localhost:8001
scoreboard_address=localhost:8000
push_address=localhost:8002

and run a scoreboard on 8000 and run jamstats on 8002 (with --scoreboardserver=localhost:8001), I get this from wsproxy:

```
level=info ts=2023-03-31T05:44:36.524175Z caller=main.go:91 pushURL=ws://localhost:8002 msg="Push connection made"
level=info ts=2023-03-31T05:44:36.524279Z caller=main.go:99 pushURL=ws://localhost:8002 msg="Push connection closed" err="websocket: close 1005 (no status)"
```

and this from jamstats:

```
Flask app started
Handshake status 200 OK
### closed: code=None, msg=None ###
```

**Comment by dhmay** [comment]
Implemented in v1.1.2
